LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -5607,5 +5610,328 @@ LocLongDescription="Free <Ability:WeaponName/> attacks on any enemies that enter
+LocPromotionPopupText = "<Bullet/> Works with squadsight. <br/> <Bullet/> <Ability:BLIND_PROTOCOL_COOLDOWN/> turn cooldown. <br/> "
+
+[LW_LeadTheTarget X2AbilityTemplate]
+LocFriendlyName = "Lead The Target"
+LocFlyOverText = "Lead The Target"
+LocHelpText = "Carefully lead the target, granting a +<Ability:LEAD_TARGET_AIM_BONUS/>% increased chance to hit, but delaying the shot until the enemies turn. Takes two actions. <Ability:LEAD_TARGET_COOLDOWN/> turn cooldown."
+LocLongDescription = "Carefully lead the target, granting a +<Ability:LEAD_TARGET_AIM_BONUS/>% increased chance to hit, but delaying the shot until the enemies turn. Takes two actions. <Ability:LEAD_TARGET_COOLDOWN/> turn cooldown."
+LocPromotionPopupText = "<Bullet/> Carefully lead the target, granting a +<Ability:LEAD_TARGET_AIM_BONUS/>% increased chance to hit, but delaying the shot until the enemies turn. Takes two actions. <Ability:LEAD_TARGET_COOLDOWN/> turn cooldown <br/>"
+
+[LW_LeadTheTargetShot X2AbilityTemplate]
+LocFriendlyName = "Lead The Target"
+LocFlyOverText = "Lead The Target"
+
+[LW_TargetFocus X2AbilityTemplate]
+LocFriendlyName = "Target Focus"
+LocHelpText = "GYour ranged attacks gain +<Ability:TARGET_FOCUS_AIM_BONUS/> aim and +<Ability:TARGET_FOCUS_PEN_BONUS/> penetration against enemies that don't take cover."
+LocLongDescription = "Your ranged attacks gain +<Ability:TARGET_FOCUS_AIM_BONUS/> aim and +<Ability:TARGET_FOCUS_PEN_BONUS/> penetration against enemies that don't take cover."
+LocPromotionPopupText = "<Bullet/> Your ranged attacks gain +<Ability:TARGET_FOCUS_AIM_BONUS/> aim and +<Ability:TARGET_FOCUS_PEN_BONUS/> penetration against enemies that don't take cover. <br/>"
+
+[LW_AimAssist X2AbilityTemplate]
+LocFriendlyName = "Aim Assist"
+LocHelpText = "Gain additional +<Ability:AIM_ASSIST_AIM_BONUS/> aim and +<Ability:AIM_ASSIST_CRIT_BONUS/> crit against holotargeted units."
+LocLongDescription = "Gain additional +<Ability:AIM_ASSIST_AIM_BONUS/> aim and +<Ability:AIM_ASSIST_CRIT_BONUS/> crit against holotargeted units."
+LocPromotionPopupText = "<Bullet/> Gain additional +<Ability:AIM_ASSIST_AIM_BONUS/> aim and +<Ability:AIM_ASSIST_CRIT_BONUS/> crit against holotargeted units. <br/>"
+
+[LW_SuperCharge X2AbilityTemplate]
+LocFriendlyName = "Arc Welder"
+LocHelpText = "You can use your arc thrower to heal friendly mechanical units."
+LocLongDescription = "You can use your arc thrower to heal friendly mechanical units."
+LocPromotionPopupText = "<Bullet/> <Ability:SUPERCHARGE_CHARGES/> Charges that heal <Ability:SUPERCHARGE_HEAL/> <br/>"
+
+[LW_ShootingSharp X2AbilityTemplate]
+LocFriendlyName = "Shooting Sharp"
+LocHelpText = "Your ranged attacks gain +<Ability:SS_PEN/> penetration against units in cover."
+LocLongDescription = "Your ranged attacks gain +<Ability:SS_PEN/> penetration against units in cover."
+LocPromotionPopupText = "<Bullet/>Your ranged attacks gain +<Ability:SS_PEN/> penetration against units in cover. <br/>"
```

Comment:
```suggestion
LocHelpText = "Your ranged attacks pierce <Ability:SS_PEN/> armor against units in cover."
LocLongDescription = "Your ranged attacks pierce <Ability:SS_PEN/> armor against units in cover."
LocPromotionPopupText = "<Bullet/>Your ranged attacks pierce <Ability:SS_PEN/> armor against units in cover. <br/><Bullet/> Does not work against unflankable units such as Drones and MECs.<br/>"
```

Reply:
Does this work against units that are in cover relative to other XCOM units, but flanked by the current shooter?